manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -124,6 +124,13 @@ spec:
               name: cni-net-dir
             - mountPath: /var/run/istio-cni
               name: cni-log-dir
+          resources:
```

Comment:
Let's add these to values and make it configurable.

Reply:
Sure